home/templates/home/home_page.html

```diff
@@ -1,15 +1,38 @@
 {% extends "base.html" %}
 {% load static %}
 {% load wagtailcore_tags %}
-
 {% block content %}
-
 <h1>A special template for the front page</h1>
 <p>
+<!-- <div class="col-sm-4"> -->
+<div class="col-md-12">
```

Comment:
This tag should be irrelevant since it defines a column that simply takes up 100% of the space?

Reply:
Only line 7 right?

Reply:
No, the `col-md-12` -- see https://getbootstrap.com/docs/5.1/layout/grid/

Reply:
I'm referring to the Line 8, which this comment is about. You can remove this line, together with the commented out line.

Reply:
I have done that and I have pushed![IMG_20220622_160218.jpg](https://user-images.githubusercontent.com/69182927/175048595-c21fbd90-f4a2-49f4-87a2-c64ec532115e.jpg)